src/Components/CreateImageWizard/steps/Snapshot/components/TemplatesEmpty.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+
+import {
+  EmptyState,
+  EmptyStateVariant,
+  EmptyStateHeader,
+  EmptyStateBody,
+  EmptyStateFooter,
+  Button,
+} from '@patternfly/react-core';
+import { ExternalLinkAltIcon } from '@patternfly/react-icons';
+
+import { TEMPLATES_URL } from '../../../../../constants';
+
+export default function TemplatesEmpty() {
```

Comment:
We're using arrow functions for components throughout the code base, could you please switch this to:
```suggestion
const TemplatesEmpty = () =>  {
```
with exporting at the end?
```
export default TemplatesEmpty;
```

Reply:
ah ok, fixed this here and in the Empty component for repos 😅